nutils/cli.py

```diff
@@ -42,6 +42,19 @@ def _relative_paths( basepath, path ):
     for i in range( len(baseparts), len(parts) ):
       yield os.path.sep.join(parts[:i]), os.path.sep.join(parts[i:])
 
+def _mkbox( text, width ):
+  lines = []
+  words = text.split()
+  while words:
+    line = words.pop(0)
+    while words and len(line) + 1 + len(words[0]) <= width:
+      line += ' ' + words.pop(0)
+    lines.append( line )
```

Comment:
Maybe this can be replaced with [textwrap]?

[textwrap]: https://docs.python.org/3.6/library/textwrap.html#textwrap.wrap